Appveyor PR Build is broken

```
Build started
git config --global --unset core.autocrlf
git clone -q https://github.com/oshi/oshi.git C:\projects\oshi
git fetch -q origin +refs/pull/[2](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L2)527/merge:
git checkout -qf FETCH_HEAD
Running Install scripts
mvn --version
Apache Maven [3](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L3).6.0 (97c98ec6[4](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L4)a1fdfee7767ce[5](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L5)ffb20918da4f719f3; 2018-10-24T18:41:47Z)
home: C:\Program Files (x8[6](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L6))\Apache\Maven\bin\..
Java version: 11.0.2, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk11
Default locale: en_US, platform encoding: Cp1252
OS name: "windows server 2016", version: "10.0", arch: "amd64", family: "windows"
java -version
java version "1.[8](https://ci.appveyor.com/project/dbwiddis/oshi/builds/48678072#L8).0_162"
Java(TM) SE Runtime Environment (build 1.8.0_162-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.162-b12, mixed mode)
appveyor-retry mvn clean package -Dmaven.test.skip=true
```
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Operating System and Hardware Information 6.4.9-SNAPSHOT:
[INFO] 
[INFO] Operating System and Hardware Information .......... FAILURE [  0.634 s]
[INFO] oshi-core .......................................... SKIPPED
[INFO] oshi-demo .......................................... SKIPPED
[INFO] oshi-core-java11 ................................... SKIPPED
[INFO] oshi-dist .......................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.880 s
[INFO] Finished at: 2023-12-04T22:50:07Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:build-helper-maven-plugin:3.5.0:parse-version (default) on project oshi-parent: The plugin org.codehaus.mojo:build-helper-maven-plugin:3.5.0 requires Maven version 3.6.3 -> [Help 1]
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -23,11 +23,11 @@ init:
   - git config --global --unset core.autocrlf
 
 install:
-  - mvn --version
+  - ./mvnw --version
   - java -version
 
 build_script:
-  - appveyor-retry mvn clean package -Dmaven.test.skip=true
+  - appveyor-retry ./mvnw clean package -Dmaven.test.skip=true
 
 test_script:
-  - mvn test
+  - ./mvnw test

